<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailBrowser</class>
 <widget class="QMainWindow" name="MailBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mail Browser</string>
  </property>
  <property name="windowOpacity">
   <double>100.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="folderListView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="messagesTableView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>781</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messageContent">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>781</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelect_email_folder"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionetm">
   <property name="text">
    <string>etm</string>
   </property>
  </action>
  <action name="actionetmadmin">
   <property name="text">
    <string>etmadmin</string>
   </property>
  </action>
  <action name="actionSelectemail_Directory">
   <property name="text">
    <string>Selectemail Directory</string>
   </property>
  </action>
  <action name="actionSelect_email_folder">
   <property name="text">
    <string>Select E-mail Folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
